programs/pluto/kernel.h

```diff
@@ -80,6 +80,12 @@ struct nic_offload {
 	enum kernel_offload_type type;
 };
 
+struct kernel_info {
+	char *name; /* eg "Linux" */
```

Comment:
Can I suggest having an enum for the OS, might as well do the streq(..., "Linux") once.